### ISSUE TITLE
Occasional crash when opening new files or reopening already opened files

### ISSUE THREAD

**Issue by Rafael2616** [AndroidIDEOfficial/AndroidIDE#1414]
### What happened?

I noticed this error in the last debug versions, when trying to open a file through the tree (I've only tried with Kotlin files) there is a small chance that the app will crash at indexOutOfBounds
This also occurs if I try to open from the open files tab
The error is the same

Occurrences: 4 times
Chance of occurring: low, but very possible 

### What's the expected behavior?

Not chash

### What version of AndroidIDE you're using?

v2.5.3 (debug builds)

### Relevant log output

```shell
AndroidIDE Crash Report
Version : v2.5.3-beta (253)
CI Build : true
Branch : dev
Commit : 3b2868120
Variant : armeabi-v7a (release)
SDK Version : 28
Supported ABIs : [armeabi-v7a, armeabi]
Manufacturer : LGE
Device : LM-X420

Stacktrace:
java.lang.IndexOutOfBoundsException: Inconsistency detected. Invalid view holder adapter positionSymbolInputAdapter$VH{6959197 position=22 id=-1, oldPos=22, pLpos:16 scrap [attachedScrap] tmpDetached no parent} com.itsaky.androidide.ui.SymbolInputView{c3fc390 GFED..... ......ID 0,0-720,70 #7f0a0294 app:id/symbol_input}, adapter:com.itsaky.androidide.adapters.SymbolInputAdapter@e72fe4d, layout:androidx.recyclerview.widget.LinearLayoutManager@e60a4ba, context:com.itsaky.androidide.activities.editor.EditorActivityKt@6a34fa0
	at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(Unknown Source:594)
	at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(Unknown Source:5)
	at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(Unknown Source:53)
	at androidx.recyclerview.widget.LinearLayoutManager.layoutChunk(Unknown Source:0)
	at androidx.recyclerview.widget.LinearLayoutManager.fill(Unknown Source:53)
	at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(Unknown Source:815)
	at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(Unknown Source:78)
	at androidx.recyclerview.widget.RecyclerView.onMeasure(Unknown Source:63)
	at android.view.View.measure(View.java:23289)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6823)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:185)
	at android.view.View.measure(View.java:23289)
	at android.widget.RelativeLayout.measureChildHorizontal(RelativeLayout.java:715)
	at android.widget.RelativeLayout.onMeasure(RelativeLayout.java:461)
	at android.view.View.measure(View.java:23289)
	at android.widget.RelativeLayout.measureChildHorizontal(RelativeLayout.java:715)
	at android.widget.RelativeLayout.onMeasure(RelativeLayout.java:461)
	at android.view.View.measure(View.java:23289)
	at com.google.android.material.bottomsheet.BottomSheetBehavior.onMeasureChild(Unknown Source:55)
	at androidx.coordinatorlayout.widget.CoordinatorLayout.onMeasure(Unknown Source:403)
	at android.view.View.measure(View.java:23289)
	at androidx.drawerlayout.widget.DrawerLayout.onMeasure(Unknown Source:245)
	at android.view.View.measure(View.java:23289)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6823)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:185)
	at androidx.appcompat.widget.ContentFrameLayout.onMeasure(Unknown Source:159)
	at android.view.View.measure(View.java:23289)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6823)
	at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1535)
	at android.widget.LinearLayout.measureVertical(LinearLayout.java:825)
	at android.widget.LinearLayout.onMeasure(LinearLayout.java:704)
	at android.view.View.measure(View.java:23289)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6823)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:185)
	at android.view.View.measure(View.java:23289)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6823)
	at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1535)
	at android.widget.LinearLayout.measureVertical(LinearLayout.java:825)
	at android.widget.LinearLayout.onMeasure(LinearLayout.java:704)
	at android.view.View.measure(View.java:23289)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6823)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:185)
	at com.android.internal.policy.DecorView.onMeasure(DecorView.java:753)
	at android.view.View.measure(View.java:23289)
	at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:2906)
	at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:1697)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2000)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1572)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:7632)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1034)
	at android.view.Choreographer.doCallbacks(Choreographer.java:845)
	at android.view.Choreographer.doFrame(Choreographer.java:780)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1020)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:205)
	at android.app.ActivityThread.main(ActivityThread.java:6993)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:884)
```

### Duplicate issues

- [X] This issue has not been reported yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Rafael2616** [comment]
Ok, this problem still occurred even in the latest debug version on November 1st

**Comment by itsaky** [comment]
Fixed in 420c95b.
